tests/unittests/RecommendationSystemTest.cpp

```diff
@@ -446,11 +504,13 @@ class RecommendationSystemTest : public BackendTest {
                          0.001);
     NodeValue bottom_MLP;
```

Comment:
Nit: Not introduced in this PR, but why names with underscores, i.e. why `bottom_MLP` instead of `bottomMLP`?

Reply:
No idea 🙂 I can change them all here since it's NFC anyway.